### PR TITLE
IT-2097: Change default CIDR for VPN

### PR DESCRIPTION
**Pull request by xschildw**
This PR changes the default VPN CIDR from 10.1.0.0/16 to 10.50.0.0/16.
Passed pre-commit.

### CODE REVIEW

File: templates/vpc.yaml [hunk 1]
```diff
@@ -19,9 +19,9 @@ Parameters:
     ConstraintDescription: List of Availability Zones in a region, such as us-east-1a, us-east-1b, us-east-1c
     Default: "us-east-1a, us-east-1b, us-east-1c"
   VpnCidr:
-    Description: CIDR of the (sophos-utm) VPN
+    Description: CIDR of the TGW VPN
     Type: String
-    Default: "10.1.0.0/16"
+    Default: "10.50.0.0/16"
```
Comment (zaro0508) [inline templates/vpc.yaml:24]: there's a duplicate static entry on line 533.  should we remove that as well?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: templates/vpc.yaml (hunk 1 above)
Comment (xschildw) [reply templates/vpc.yaml:24]: You mean the  !Ref? That should be fine shouldn't it (just change from 10.1.* to 10.50.*)?

Comment (zaro0508) [reply templates/vpc.yaml:24]: No I meant this section..
```
      SecurityGroupIngress:
        - CidrIp: !Ref VpnCidr
          FromPort: -1
          ToPort: -1
          IpProtocol: "-1"
          Description: "Allow all VPN traffic"
        - CidrIp: "10.50.0.0/16"    # AWS TGW Hub
          FromPort: -1
          ToPort: -1
          IpProtocol: "-1"
          Description: "Allow AWS TGW Hub traffic"
```

now that you've changed `!Ref VpnCidr` to a default of `10.50.0.0/16` isn't it going to be the same as the `AWS TGW Hub`?  Therefore shouldn't we just git rid of the static (AWS TGW Hub) CidrIp config?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: templates/vpc.yaml (hunk 1 above)
Comment (xschildw) [reply templates/vpc.yaml:24]: Indeed...

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**